email_handler.py

```diff
@@ -555,6 +555,16 @@ def handle_reply(envelope, smtp: SMTP, msg: Message, rcpt_to: str) -> (bool, str
         envelope.rcpt_options,
     )
 
+    # replace the "ra+string@simplelogin.co" by the alias in the email body
+    # as this is usually included in when replying
+    if user.replace_reverse_alias:
+        payload = (
+            msg.get_payload()
+            .encode()
+            .replace(reply_email.encode(), contact.website_email.encode())
```

Comment:
Nice spot! I don't know why I didn't realize that it doesn't make sense to replace the `reply_email` by the `alias` before ...